src/main/java/com/mraof/minestuck/block/MSBlocks.java

```diff
@@ -253,6 +253,7 @@ private static Block coalOre(BlockBehaviour.Properties properties)
 	public static final RegistryObject<Block> DEAD_LOG = REGISTER.register("dead_log", () -> new FlammableLogBlock(Block.Properties.of(Material.WOOD, logColors(MaterialColor.WOOD, MaterialColor.PODZOL)).strength(2.0F).sound(SoundType.WOOD)));
 	public static final RegistryObject<Block> PETRIFIED_LOG = REGISTER.register("petrified_log", () -> new RotatedPillarBlock(Block.Properties.of(Material.WOOD, logColors(MaterialColor.WOOD, MaterialColor.PODZOL)).strength(2.0F).sound(SoundType.STONE)));
 	public static final RegistryObject<Block> GLOWING_WOOD = REGISTER.register("glowing_wood", () -> new FlammableLogBlock(Block.Properties.of(Material.WOOD, MaterialColor.COLOR_LIGHT_BLUE).strength(2.0F).lightLevel(state -> 11).sound(SoundType.WOOD)));
+	public static final RegistryObject<Block> SHADE_WOOD = REGISTER.register("shade_wood", () -> new FlammableLogBlock(Block.Properties.of(Material.WOOD, MaterialColor.COLOR_LIGHT_BLUE).strength(2.0F).lightLevel(state -> 11).sound(SoundType.WOOD)));
```

Comment:
Select and right click SHADE_WOOD, find the option labeled Refactor, and inside of that select the option called Rename. Then rename it to SHADEWOOD_LOG. This will change every instance of SHADE_WOOD in code to SHADEWOOD_LOG. 

Also change the registry name from shade_wood to shadewood_log as well. There is no refactoring option for this one, so manually go to the blockstate/model/texture files and change the name of the files and the contents of those files to reflect the switch to shadewood_log. 

Then here in MSBlocks, and the other locations where SHADE_WOOD is used, move this to be with the logs instead of the woods

This is all for reasons that are explained in the encompassing review comment

Reply:
Also, since this block doesnt have any glowing elements anymore, remove the light level addition